<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>/*
 * Copyright Â© 2016 Artem Smirnov
 * All right reserved.
 * Author: Artem Smirnov &lt;msvrtm@gmail.com&gt;
 * License: http://www.gnu.org/licenses/gpl.txt
 */

</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LCD Font Converter</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="CharacterInfoTableView" name="characterInfoTableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="updateTable">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTabWidget" name="tabWidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>220</height>
           </size>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="FixedCharactersPage" name="fixedCharacters">
           <attribute name="title">
            <string>Fixed characters</string>
           </attribute>
          </widget>
          <widget class="SelectedCharactersPage" name="selectedCharacters">
           <attribute name="title">
            <string>Selected characters</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="CharacterPropWidget" name="characterPropWidget" native="true"/>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="backgroundBrush">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>232</red>
             <green>231</green>
             <blue>231</blue>
            </color>
           </brush>
          </property>
          <property name="renderHints">
           <set>QPainter::NonCosmeticDefaultPen</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>F&amp;ont</string>
    </property>
    <addaction name="actionImportFont"/>
    <addaction name="actionExportFontC"/>
    <addaction name="separator"/>
    <addaction name="actionNoAntialias"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFont"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImportFont">
   <property name="text">
    <string>&amp;Import existing system font</string>
   </property>
  </action>
  <action name="actionExportFontC">
   <property name="text">
    <string>&amp;Export the font as C file</string>
   </property>
  </action>
  <action name="actionNoAntialias">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;NoAntialias</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CharacterPropWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/characterPropertyWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectedCharactersPage</class>
   <extends>QWidget</extends>
   <header>Widgets/Tab/selectedCharactersPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharacterInfoTableView</class>
   <extends>QTableView</extends>
   <header>Widgets/TableView/characterInfoTableView.h</header>
  </customwidget>
  <customwidget>
   <class>FixedCharactersPage</class>
   <extends>QWidget</extends>
   <header>Widgets/Tab/fixedCharactersPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
